umbraco-cloud/security.md

```diff
@@ -120,14 +120,16 @@ The following rule can be added to your web.config file in the `system.webServer
 Please note these two different variations, which differ if you have a reverse proxy like Cloudflare (with Proxying turned on) in front of the website.
 
 {% hint style="info" %}
-Since December 8th, 2020 all Umbraco Cloud sites uses Cloudflare for DNS, so new and updated projects should use the Reverse Proxy version.
+Since December 8th, 2020 all Umbraco Cloud sites uses Cloudflare for DNS, so new and updated projects should use the reverse proxy version.
 
 If you are unsure whether your Cloud project uses Cloudflare or not, get in touch with the friendly support team, and they will help you out.
 {% endhint %}
 
-**Reverse Proxy version (eg. Cloudflare)**
+{% tabs %}
+{% tab title="Default (reverse proxy)" %}
+**Default setup (reverse proxy version, eg. Cloudflare)**
 
-When using Cloudflare, which is the default setup for all Cloud projects, the project will from behind a reverse proxy get the IPs from the `CF-Connecting-IP` header. In this case, which is most cases, use the first variation here to restrict access to your backoffice using IP filtering.
+For all cloud projects since December 8th, 2020, Cloudflare is used as a reverse proxy service for your project. This means that you'll need to get the IPs from the `CF-Connecting-IP` header. Use the example below to restrict access to your backoffice using IP filtering.
```

Comment:
This date is already mentioned in the note above the tabs.
Perhaps we could rephrase this a bit:

"By default, Cloudflare is used as a reverse proxy service on all projects. This means [...]"

Reply:
Updated the PR :-) Changed it a little bit, as another by default felt a bit too excessive, hah.